FlashEnglish/Sources/Views/Home/HomeView.swift

```diff
@@ -8,22 +8,27 @@
 import SwiftUI
 
 struct HomeView: View {
+
+
+
+
+
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/Masaya1582/FlashEnglish/blob/61903b517db35dae022b02953232898fb7ba090e/FlashEnglish/Sources/Views/Home/HomeView.swift#L15"></span>Limit vertical whitespace to a single empty line; currently 5<br />
<code>vertical_whitespace</code> <code>HomeView.swift:15</code></td>
    </tr>
  </tbody>
</table>